### ISSUE TITLE
A question about the total number of 3D scene point cloud

### ISSUE THREAD

**Issue by RongkunYang** [scannetpp/scannetpp#91]
Hi, dear authours. Thank you for the wonderful datasets.
After I download all the 3D point cloud, there seems to be 330 3D point cloud scenes, but I read the paper of ScanNet++, it is mentioned that there are a total of 460 scenes.
Have I not downloaded some of the point cloud scenes?

Thank you.

**Comment by cy94** [comment]
Hello @RongkunYang thanks for your interest. The details of the first data release are here https://kaldir.vc.in.tum.de/scannetpp/documentation along with the number of scenes. 
We'll be releasing the remaining scenes + more soon, stay tuned!

**Comment by RongkunYang** [comment]
OK, thank you for your fast reply. 
I have another question, may I ask whether the processing code for 3D instance segmentation task can be shared, such as the processing code for SoftGroup. Or I need follow the processing step on this repository can obtain relevant data format

**Comment by cy94** [comment]
Hi, the data preparation scripts are in this repo. You can adapt it to whichever code you like. We haven't released the baseline training code yet

**Comment by RongkunYang** [comment]
I got it. 
When preparing the semantic and instance segmentation data, on the step of "Prepare Semantic/Instance Ground Truth Files for Evaluation".
It is mentioned "but without point sampling",  this means that we use the mesh vertices as point cloud for evaluation, and we need comment out "- sample_points_on_mesh" in the **transforms** of **prepare_training_data.yml**, right?
![image](https://github.com/scannetpp/scannetpp/assets/71752017/8770cb0f-cbaf-4b53-8f07-0830aca37cbe)

**Comment by cy94** [comment]
The final evaluation on the test set is done on the vertices. You can sample points and then transfer the predictions to the vertices, or directly make predictions on the vertices. The first one is recommend to avoid any problems with voxelization
